github/AdvisoryCredit.py

```diff
@@ -1,6 +1,6 @@
 ############################ Copyrights and license ############################
 #                                                                              #
-# Copyright 2023 Jonathan Leitschuh <Jonathan.Leitschuh@gmail.com>             #
```

Comment:
removing this line does not look right either

Reply:
Same issue here, this file was renamed so the script removed it.